crates/gen/src/gen/interface.rs

```diff
@@ -50,34 +47,41 @@ impl Interface {
         dependencies
     }
 
-    pub fn to_tokens(&self) -> TokenStream {
-        let definition = self.name.to_definition_tokens();
-        let abi_definition = self.name.to_abi_definition_tokens();
-        let name = &self.name.tokens;
-        let phantoms = self.name.phantoms();
-        let constraints = &self.name.constraints;
-
-        let default_interface = self
-            .interfaces
+    pub fn default_interface(&self) -> &gen::RequiredInterface {
+        self.interfaces
             .iter()
             .find(|i| i.kind == InterfaceKind::Default)
-            .unwrap();
+            .unwrap()
+    }
+
+    pub fn gen_vtable_initializer(&self) -> TokenStream {
+        panic!();
```

Comment:
nit: "For work-in-progress" items: the `todo!()` macro is much more descriptive than using an empty `panic!`.